src/BlazingPizza.MenuService/appsettings.json

```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "ConnectionStrings": {
+    "MenuDatabase":"Server=sql;Database=menu;User Id=sa;Password=yourStrong(!)Password;"
+  },

```

Comment:
!!!!!!!!!!!

Reply:
This should be done by tye. If not that, then done by docker compose with an env-var